index.html

```diff
@@ -23,6 +23,7 @@
   <script src="https://code.jquery.com/jquery-3.2.1.min.js" integrity="sha256-hwg4gsxgFZhOsEEamdOYGBf13FyQuiTwlAQgxVSNgt4=" crossorigin="anonymous"></script>
   <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
   <script src="https://use.fontawesome.com/a26e812ff2.js"></script>
+  <link href="//netdna.bootstrapcdn.com/font-awesome/3.2.1/css/font-awesome.css" rel="stylesheet">
```

Comment:
Why do you need this? On the line above it imports fontawesome with some fancy metrics and CDN magic I don't understand or need.

Reply:
To be honest, I don't really know why it needs both, but it seems like one covers some symbols and the other covers the rest.

Reply:
Any idea of what symbols? I am running 4.7, it looks like you're trying to import version 3.2?

Reply:
The original doesn't cover the arrows. I'll check 4.7 for arrows that would work as a substitute.

Reply:
Ok, it seems that they updated the name of the arrows so I fixed them and removed the import.